### PR TITLE
[Dropdown] Multiple search keyboard + icon fix

### PR DESCRIPTION
**Pull request by lubber-de**
## Description

A multiple dropdown selection searchable box had

1. The close icon displayed over the search input field when many entries have been selected
2. Whenever an entry was selected via enter, it looses it's focus making quick keyboard selection impossible (you needed to click again into the search field

Before:
multidd_before

After:
multidd_after

### CODE REVIEW

File: dist/components/dropdown.js [hunk 1]
```diff
@@ -1451,6 +1451,7 @@ $.fn.dropdown = function(parameters) {
                     module.event.item.click.call($selectedItem, event);
                     if(module.is.searchSelection()) {
                       module.remove.searchTerm();
+                      $search.focus();
```
Comment (hammy2899) [inline dist/components/dropdown.js]: Why is this line added but the src not provided?

Comment (lubber-de) [reply dist/components/dropdown.js]: I am confused.. Which source is missing?

Comment (hammy2899) [reply dist/components/dropdown.js]: This is the dist file

**Comment by ColinFrick** [comment]
Thanks for the pull request. Just a quick note, you shouldn't include the dist files in the commits, that would make merging easier :)

File: dist/components/dropdown.js (hunk 1 above)
Comment (lubber-de) [reply dist/components/dropdown.js]: :fearful: My bad...now it's there.  :smile:

**Comment by lubber-de** [comment]
> Thanks for the pull request. Just a quick note, you shouldn't include the dist files in the commits, that would make merging easier :)

Ok, sorry, i'll keep that in mind from now on
